File: backend/Entities/Models/FilialLastMile.cs
using System;
using System.Collections.Generic;

namespace Entities.Models;

public partial class FilialLastMile : BaseEntity
{
    public Guid Id { get; set; }

    public int FilialId { get; set; }

    public string? CepInicial { get; set; }

    public string? CepFinal { get; set; }

    public int? BairroId { get; set; }

    public int? MunicipioId { get; set; }

    public int? EstadoId { get; set; }

    public DateTime? DataCadastro { get; set; }

    public virtual Bairro? Bairro { get; set; }

    public virtual Estado? Estado { get; set; }

    public virtual Filial Filial { get; set; } = null!;

    public virtual ICollection<FilialLastMileGrupoItem> FilialLastMileGrupoItems { get; } = new List<FilialLastMileGrupoItem>();

    public virtual Municipio? Municipio { get; set; }
}

File: frontend-with-menus/src/app/Business/Models/filialLastMile.ts
import { Municipio } from "./Municipio";
import { Bairro } from "./bairro";
import { Estado } from "./estado";
import { Filial } from "./filial";
import { FilialLastMileGrupoItem } from "./filialLastMileGrupoItem";

export interface FilialLastMile  {
    id: string;
    filialId: number;
    cepInicial: string | null;
    cepFinal: string | null;
    bairroId: number | null;
    municipioId: number | null;
    estadoId: number | null;
    dataCadastro: string | null;
    bairro: Bairro | null;
    estado: Estado | null;
    filial: Filial;
    filialLastMileGrupoItems: FilialLastMileGrupoItem[];
    municipio: Municipio | null;
}